wapitiCore/attack/mod_cms.py

```diff
@@ -0,0 +1,79 @@
+import re
+import json
+from os.path import join as path_join
+
+from wapitiCore.net.web import Request
+from wapitiCore.attack.attack import Attack
+from wapitiCore.language.vulnerability import LOW_LEVEL, _
+from wapitiCore.definitions.fingerprint import NAME as TECHNO_DETECTED
+
+MSG_TECHNO_VERSIONED = _("{0} {1} detected")
+MSG_NO_WP = _("No WordPress Detected")
+class mod_cms(Attack):
+    """Detect WordPress Plugins with version."""
+    name = "cms"
+    PAYLOADS_FILE = "wordpress_plugins.txt"
+
+    def get_plugin(self):
+        with open(path_join(self.DATA_DIR, self.PAYLOADS_FILE), errors="ignore") as plugin_list:
+            for line in plugin_list:
+                plugin= line.strip()
+                if plugin:
+                    yield plugin
+
+    def detect_plugin(self, url):
+        for plugin in self.get_plugin():
+            req= Request('{}/wp-content/plugins/{}/readme.txt'.format(url, plugin))
```

Comment:
Can you please fix whitespaces around operators to comply with PEP8 recommandations ? https://www.python.org/dev/peps/pep-0008/#other-recommendations